README.md

```diff
@@ -147,6 +147,7 @@ const portuguese = voices.portuguese()
 const dutch = voices.dutch()
 const russian = voices.russian()
 const german = voices.german()
+const mexican = voices.mexican()
```

Comment:
this should be in the `language and country` block because it's related to a specific country :)

Reply:
Done